### DISCUSSION TITLE
Remove the first PC?

### DISCUSSION THREAD

**Discussion by brianpenghe** [GreenleafLab/ArchR#1087] [Questions / Documentation] [answered]
I saw that the tutorial uses the top 30 PCs for downstream analysis.
I was wondering whether it makes more sense to remove the first PC which is usually correlated to data quality.
Is there an easy way to check whether the first PC is correlated to quality?
Can we plot PC1's scores on UMAP etc. ?

**Comment by rcorces** [comment, marked as answer]
ArchR handles this automatically. see the `corCutOff` argument to various downstream functions.
https://www.archrproject.com/reference/addUMAP.html

**Reply by brianpenghe** [reply]
That's so smart :)
